force-app/main/userCommunity/lwc/hot_requestList/hot_requestList.html

```diff
@@ -96,8 +96,8 @@ <h2 style="font-size: 1.25rem">Informasjon om virksomhet</h2>
 
                 <section class="bestilling-info-skjema sub-form">
                     <h2 style="font-size: 1.25rem">Informasjon om tolkebehov</h2>
-
                     <c-hot_record-details record={requestFields}></c-hot_record-details>
+                    <c-hot_record-files record-id={recordId}></c-hot_record-files>
```

Comment:
OBS! Bestiller skal ikke se denne i første omgang. Vi må legge til et filter for at det er brukeren som klikker på detaljer.
kan sjekke Request.Account__c == this.userRecord.AccountId, og sette en track parameter, som du bruker til å skjule/vise recordFiles

Reply:
Trengs dette på rediger og?

Reply:
Tenker det ja. Er vel bare Bestiller (bruker om bestiller til seg selv) som kan klikke rediger?

Reply:
Det er allerede good for rediger, ser jeg.
`editOrder(row) {
        const { Id } = row;
        const index = this.findRowIndexById(Id);
        if (index !== -1) {
            if (row.Orderer__c === this.userRecord.AccountId) {
                if (this.requests[index].ExternalRequestStatus__c.includes('Åpen')) {
                    //Here we should get the entire record from salesforce, to get entire interpretation address.
                    let clone = this.requests[index];
                    this[NavigationMixin.Navigate]({
                        type: 'comm__namedPage',
                        attributes: {
                            pageName: 'ny-bestilling'
                        },
                        state: {
                            fieldValues: JSON.stringify(clone),
                            fromList: true,
                            edit: true
                        }
                    });
                }
            } else {
                alert('Denne bestillingen er bestilt av noen andre, og du har ikke rettigheter til å endre den.');
            }
        }`